packages/integration-tests/__tests__/defined-behavior/2-expected-use/__snapshots__/custom-operations.ts.snap

```diff
@@ -40,6 +40,86 @@ exports[`custom operations async sync 1`] = `
 ]
 `;
 
+exports[`custom operations client operations with custom types and refs client can call custom mutation with custom type argument 1`] = `
+[
+  [
+    {
+      "authMode": undefined,
+      "authToken": undefined,
+      "query": "
+    mutation($customArg: ID) {
```

Comment:
Here and below, `ID` is not the correct GraphQL data type for these operations. We should be referencing the Input type that is generated from our Custom Type